### ISSUE TITLE
Hotkeys can be pressed in forced window mode

### ISSUE THREAD

**Issue by peoplearestrange** [tgstation/tgstation#47919]
## Round ID:
124143 Events EU

## Reproduction:
Whilst in command bar forced mode (pressing tab until its highlighted red)
Click mouse on game or buttons area then press a key within roughly 1 second
If key is a hotkey it will be pressed

I.e.
Tab to Focus bar
Click and then press O
OOC dialogue box is opened up and that letter will be removed from command bar, but command bar will still be the active area.

Basically you loose a keystroke and you get an unwanted dialogue box opening up if you press something like O or T

**Comment by Couls** [comment]
I don't think there's anything I can do about this, the delay of roughly a second can't be avoided

**Comment by kriskog** [comment]
@Couls This is a result of the hack to make it more classic-like?

**Comment by Couls** [comment]
Yeah the winset is why there's a delay
